src/rdkafka_coord.c

```diff
@@ -403,9 +403,6 @@ static void rd_kafka_coord_req_handle_FindCoordinator(rd_kafka_t *rk,
             RD_KAFKA_ERR_ACTION_RETRY,
             RD_KAFKA_RESP_ERR_COORDINATOR_NOT_AVAILABLE,
 
-            RD_KAFKA_ERR_ACTION_RETRY,
```

Comment:
Should we remove the same action,error tuple from `rd_kafka_handle_OffsetCommit_error` inside src/rdkafka_request.c? Given that it is redundant now too.

Reply:
We can keep it, for clarity.